auto_updater.sh

```diff
@@ -0,0 +1,84 @@
+#!/bin/bash set -eu
+
+function check_fork {
+  git ls-remote --exit-code fork > /dev/null 2>&1
```

Comment:
Why is this remote repo called `fork`?  Isn't it `origin` since it is `git@github.com:adoptopenjdk-github-bot/homebrew-openjdk.git`?

Reply:
nope fork is good, it's the fork of the upstream (helps differentiate)